### ISSUE TITLE
Cannot enable module in version 10.3

### ISSUE THREAD

**Issue by thebrownjumper** [deidril/pf1-pdf-en-import#4]
When I try to enable the module in version 10.3 I see...

<img width="248" alt="image" src="https://github.com/deidril/pf1-pdf-en-import/assets/81319107/151b041d-b34b-42b9-8ca1-cb6602803e1b">

**Comment by Trellinane** [comment]
I am getting the same error even though version 3.04 indicates it is verified for PF1E 10.3.

**Comment by deidril** [comment]
It seems v10 broke the module. The import module has a dependency to the PF1 Bestiary module. 
Looks like a lot of work. Not sure if i should wait bestiary module is fixed or not.

**Comment by thebrownjumper** [comment]
PF1E v11.0 is now available... Still hopeful 🥇

**Comment by deidril** [comment]
Yes, indeed. I'm starting to work on the migration to foundry v12.

**Comment by deidril** [comment]
Released a foundry v12 and pf1 system v11 compatible version.
